### ISSUE TITLE
BasicUI: valuecolor not updated

### ISSUE THREAD

**Issue by rtvb** [eclipse/smarthome#3065]
Any time the color has changed to match a value, it will not revert back to the default color (for unspecified ranges).

```php
Text item=System_CPU_Load valuecolor=[System_CPU_Load=="NULL"="lightgray",System_CPU_Load>90="red",>40="orange"]
```

I did some testing. As long as the value stays below the treshold of 40, it has the default color. When it passes the threshold of 40 it becomes orange. But when it then lowers back below the threshold (say 10%) it remains orange and never turns back to the default color. 

Origin: https://github.com/openhab/openhab-docs/issues/316

**Comment by lolodomo** [comment]
I was able to reproduce it. It only concerns Basic UI.
I will try to fix it.

**Comment by lolodomo** [comment]
https://github.com/eclipse/smarthome/blob/master/extensions/ui/org.eclipse.smarthome.ui.basic/web-src/smarthome.js#L1646

@resetnow : at this place, it is necessary, when label color or value color is undefined to call setLabelColor / setValueColor with the default color. How getting the default color ?

**Comment by resetnow** [comment]
@lolodomo it's not implemented right now, you can extend Control function to save default colors from either computed style or data attribute (better). setLabelColor/setValueColor could accept undefined values then (which will revert the color to default). Also, these functions should check if the current color differs from the one being assigned (to prevent interactions with DOM when it's not needed)

**Comment by resetnow** [comment]
(I was replying to comment which was probably deleted)

**Comment by lolodomo** [comment]
@resetnow : I suppressed my message because I found the solution.
I just submitted a PR that fixes the issue.

**Comment by lolodomo** [comment]
To be closed now that PR #3139 is merged.
